src/client/Microsoft.Identity.Client/Platforms/uap/WamBroker/WamBroker.cs

```diff
@@ -87,7 +72,7 @@ internal class WamBroker : IBroker
             _msaPassthroughHandler = msaPassthroughHandler ??
                 new MsaPassthroughHandler(_logger, _msaPlugin, _wamProxy, _parentHandle);
 
-            _wamOptions = appConfig.WindowsBrokerOptions ??
+            _wamOptions = appConfig.UwpBrokerOptions ??
```

Comment:
UWP / UAP?

Reply:
UWP is the latest name of Uap. So UWP is better. However, I would not invest time in changing from Uap to UWP.